packages/dd-trace/src/appsec/sdk/track_event.js

```diff
@@ -107,5 +224,9 @@ function runWaf (eventName, user) {
 module.exports = {
   trackUserLoginSuccessEvent,
   trackUserLoginFailureEvent,
-  trackCustomEvent
+  trackUserLoginSuccessV2,
+  trackUserLoginFailureV2,
+  trackCustomEvent,
```

Comment:
i made a mistake in my previous suggestion
```suggestion
  trackCustomEvent,
  trackUserLoginSuccessV2,
  trackUserLoginFailureV2,
```